### PR TITLE
Add NiftiMasker ref to signals_to_img_labels

### LINKED ISSUE #1753: add Seealsos: refs to NiftiMasker
I would vote to put see also refs to NiftiMasker

in regions functions eg in:

https://nilearn.github.io/modules/generated/nilearn.regions.img_to_signals_labels.html#nilearn.regions.img_to_signals_labels

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes #1753

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. The travis failure seems unrelated.

File: nilearn/regions/signal_extraction.py [hunk 1]
```diff
@@ -151,6 +151,7 @@ def signals_to_img_labels(signals, labels_img, mask_img=None,
     --------
     nilearn.regions.img_to_signals_labels
     nilearn.regions.signals_to_img_maps
+    nilearn.input_data.NiftiMasker
```
Comment (KamalakerDadi) [inline nilearn/regions/signal_extraction.py]: This function is more useable by users who wants to extract signals based on labels in atlases. I would put also ```nilearn.input_data.NiftiLabelsMasker : Signal extraction on labels images e.g. clusters```.

I would suggest to add also few details about what nilearn.input_data.NiftiLabelsMasker is ?
You can inspire from this reference:
https://github.com/nilearn/nilearn/blob/master/nilearn/plotting/img_plotting.py#L677

**Comment by KamalakerDadi** [comment]
> In general, the maskers should be mentioned more as "see also"s in this module, as people find this >module naturally instead of the maskers. 

Given this comment in the issue tracker #1753 . We should also add See also in function ```img_to_signals_maps``` like what you did for ```img_to_signals_labels```

File: nilearn/regions/signal_extraction.py (hunk 1 above)
Comment (KamalakerDadi) [reply nilearn/regions/signal_extraction.py]: You can skip my comment here. NiftiMasker is enough I think.

**Comment by KamalakerDadi** [comment]
Things which have missed to add the maskers family are:
- img_to_signals_labels where we need to add about ```nilearn.input_data.NiftiLabelsMasker : Signal extraction on labels images e.g. clusters```
- img_to_signals_maps  where we need to add about ```nilearn.input_data.NiftiMapsMasker : Signal extraction on probabilistic maps e.g. ICA```
- ```nilearn.input_data.NiftiMasker``` should be added in signals_to_img_maps.

Minor comments.

**Comment by souravsingh** [comment]
Thanks for the comments. I will look into it and fix the PR.

**Comment by GaelVaroquaux** [comment]
Any news on this PR?

File: nilearn/regions/signal_extraction.py [hunk 2]
```diff
@@ -303,6 +309,7 @@ def signals_to_img_maps(region_signals, maps_img, mask_img=None):
     --------
     nilearn.regions.signals_to_img_labels
     nilearn.regions.img_to_signals_maps
+    nilearn.input_data.NiftiMasker
```
Comment (GaelVaroquaux) [inline nilearn/regions/signal_extraction.py]: It should be NiftiMapsMasker here.

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thank you very much!

**Pull request merged**